devicetypes/smartthings/zwave-motion-light-sensor.src/zwave-motion-light-sensor.groovy

```diff
@@ -89,6 +90,10 @@ def updated() {
 def configure() {
 	// Device wakes up every deviceCheckInterval hours, this interval allows us to miss one wakeup notification before marking offline
 	sendEvent(name: "checkInterval", value: 2 * deviceWakeUpInterval * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+
+	if (isEverspring()) {
```

Comment:
The more I think, I rather make this device specific.

Reply:
@PKacprowiczS I'll merge once you make this change. As Kianoosh points out, since it's a specific configuration parameter and those are usually device-specific, it's probably best to make this model-specific.

Reply:
@kianooshST @greens done